components/infobox/commons/infobox_team.lua

```diff
@@ -169,13 +169,17 @@ function Team:_createLocation(location)
 			'[[:Category:' .. location .. '|' .. location .. ']]'
 end
 
+function Team:_getStandardLocationValue(location)
+	return Flags.CountryName(location) or location
```

Comment:
If I'm not reading Flags.CountryName() wrong, it can never return nil or false (should only return empty string or normalized named)
so the `or location` is entirely redundant?

Reply:
yeah, this should be okay though:
```suggestion
	local standardizedLocation = Flags.CountryName(location)
	return standardizedLocation ~= '' and standardizedLocation or location
```

Reply:
I disagree on saving a location that can't be found.
I'm thinking if we should add a warning (similar the legacy bracket warning) and then saving empty string to the database. Imo we shouldn't populate the database with bad values.

Reply:
okay for not storing it
idk about the warning, we already set a tracking category (`Pages with unknown flags`) for that case (via `Flags.Icon` in `Team:_createLocation`)

Reply:
If a wiki overrides the Location Display, that tracking category won't be set?
Also a tracking category requires someone with knowledge of it to check for it, on every wiki, periodically. 
A warning will show editors directly that something is wrong and should be addressed, be it a typo, a missing country in masterdata or something else.

Reply:
> Imo we shouldn't populate the database with bad values.

100%.